docs/book/src/migration/migration_guide_v2tov3.md

```diff
@@ -0,0 +1,239 @@
+# Migration from v2 to v3
+
+Make sure you understand the [differences between Kubebuilder v2 and v3](./v2vsv3.md)
+before continuing
+
+Please ensure you have followed the [installation guide](/quick-start.md#installation)
+to install the required components.
+
+## Steps to migrate to v3 project layout
+
```

Comment:
You are saying to add and modify fields without saying where those fields need to be added or updated.

> The following changes need to be applied to the project configuration file (`PROJECT`) that can be found at the root directory: